1908-01-20.xml

```diff
@@ -0,0 +1,5143 @@
+                        Dragoons v. the' World.</p>
+                    <p>Tues. 21 K, S. C. 60th Rifles Band. Polo Matches. Mr. Schpeiber's team v.
+                        K.8. ©.</p>
+                    <p>Tues. 29 Shepheard's Hofsl, Annual Grand Ball</p>
+                    
+                </div>
+                <div type="section" feature="prepaidAdvertisements">
+                    <head>Cheap Prepaid Advertisements</head>
+                    <p>Under this heading advertisements are inserted at the following rates :—</p>
+                    <table rows="4" cols="4">
+                        <row role="label">
+                            <cell/>
+                            <cell>ONCE</cell>
+                            <cell>3 TIMES</cell>
+                            <cell>6 TIMES</cell>
+                        </row>
+                        <row>
+                            <cell>15 words . . .</cell>
+                            <cell>P.T. 5</cell>
+                            <cell>P.T. 10</cell>
+                            <cell>P.T. 15</cell>
+                        </row>
+                        <row>
+                            <cell>30 words ...</cell>
+                            <cell>„ 8</cell>
+                            <cell>„ 16</cell>
+                            <cell>„ 24</cell>
+                        </row>
+                        <row>
+                            <cell>Every 10 words, beyond 30. . .</cell>
+                            <cell>,, 2</cell>
+                            <cell>,, 4</cell>
+                            <cell>,, 6</cell>
+                        </row>
+                    </table>
+                    <p>The address is counted. The advertisement must appear on consecutive days for
+                        above rates to be obtained. 50% extra is charged for advertisements not
+                        appearing consecutively.</p>
+                    <p>All such advertisements must be prepaid, and to this rule no exception
+                        whatever will be made. Letters in reply to advertisements will be posted to
+                        any address if a few stamps are sent by the advertiser to cover postage.</p>
+                    <div type="item">
+                        <p>AGARD'S INTERNATIONAL INDUSTRIES AND TRADE MARKS REGISTER. – A useful
+                            business directory containing addresses of all important business firms of
+                            Great Britain the Continent, and Egypt. Circulating all over Europe and
+                            America. Price— One pound Sterling. Post Free.</p>
+                        <p>AGARD'S INTERNATIONAL HOTEL GUIDE sent post free to all first class
+                            Hotels throughout Europe, America, the Colonies and Egypt. The best
+                            reference book for travellers.</p>
+                        
+                    </div>
+                </div>
```

Comment:
move this `</div>` to the end of the prepaid advertisements section, so that all of the items are within the section.